src/docs/perf/rendering/shader.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Reduce shader compilation jank on mobile
+short-title: Shader jank
+description: What is shader jank and how to minimize it.
+---
+
+If the animations on your mobile app appear to be janky,
+but only on the first run, you can _warm up_ the
+shader for the Skia Shader Language (SkSL) for a
+significant improvement.
+
+![Side-by-side screenshots of janky mobile app next to non-janky app]({% asset perf/render/shader-jank.gif @path %}){:width="100%"}
+
+## What is shader compilation jank?
+
+If an app has janky animations during the first run,
+and later becomes smooth for the same animation,
+then it's very likely due to shader compilation jank.
+
+More technically, a shader is a piece of program that runs on
+a GPU (graphics processing unit).
+When a shader is first used, it needs to be compiled.
+The compilation could cost up to a few hundred milliseconds
+whereas a smooth frame needs to be drawn within 16 milliseconds
+for a 60 fps (frame-per-second) display.
+Therefore, a compilation could cause tens of frames to be missed,
+and drop the fps from 60 to 6. This is _compilation jank_.
```

Comment:
> drop the fps from 60 to 6

Well, doesn't this make the problem look worse than it is? I mean, compilation jank is bad, but it's not like we're compiling on every frame. We typically have several missed frames, but after that, the app is smooth again.

This almost looks like the app starts on a consistent 6 FPS framerate for some time, which, I believe, is not the case.

Reply:
Shall we  change `drop the fps from 60 to 6` to `drop the fps from 60 to 6 for a short duration in the first run`?

Reply:
I added a follow on sentence saying it's smooth after.